src/commands/dev/dev.js

```diff
@@ -522,11 +561,26 @@ const createDevCommand = (program) => {
         .hideHelp(),
     )
     .addOption(new Option('--graph', 'enable Netlify Graph support').hideHelp())
+    .addOption(
+      new Option('-e, --edgeInspect [inspectHostPort]', 'enable Deno inspect with optional inspectHostPort').conflicts(
```

Comment:
I made a couple of suggestions here. First, we're using `<parameter>` instead of `[parameter]`. I also think we can simplify `inspectHostPort` to just `address` and then use the description to explain the format.

Finally, I've reworded things a bit, let me know what you think.

```suggestion
      new Option('-e, --edgeInspect <address>', 'enable the V8 Inspector Protocol for Edge Functions, with an optional address in the host:port format').conflicts(
```

Reply:
I think square brackets indicate an optional argument, whereas `<>` indicates required. At least looking I see this in the docs: https://tj.github.io/commander.js/#Command.prototype.option

Reply:
Ah, you're right! My bad.